modules/exploits/windows/fileformat/videocharge_studio.rb

```diff
@@ -0,0 +1,92 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  include Msf::Exploit::FILEFORMAT
+  include Msf::Exploit::Seh
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'    => 'VideoCharge Studio 2.12.3.685 Buffer Overflow (SEH)',
```

Comment:
We try to avoid having software version information included in the module title.